examples/modal/src/App.tsx

```diff
@@ -0,0 +1,218 @@
+import * as React from "react";
+import {
+  Routes,
+  Route,
+  Outlet,
+  Link,
+  useLocation,
+  useNavigate,
+  useParams
+} from "react-router-dom";
+
+export default function App() {
+  let location = useLocation();
+
+  // This piece of state is set when one of the
```

Comment:
I've been formatting my examples using 80-char printWidth in prettier. Let's use that everywhere.